src/main/java/com/zendesk/maxwell/RowMap.java

```diff
@@ -105,7 +105,7 @@ private String pkToJsonHash() throws IOException {
 				if ( data.containsKey(pk) )
```

Comment:
wasn't this check working before, or does `pk` need to be lower case here as well?